src/redist/targets/packaging/windows/dotnetCoreUninstall/registrykeys.wxs

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
There's nothing here - delete the file and reference.

Reply:
removed